src/ray/raylet/local_object_manager.cc

```diff
@@ -265,6 +295,86 @@ void LocalObjectManager::AsyncRestoreSpilledObject(
   });
 }
 
+void LocalObjectManager::ProcessSpilledObjectsDeleteQueue(uint32_t max_batch_size) {
```

Comment:
Can we combine this method into the existing `FlushFreeObjects()` method?

Reply:
I removed the periodic operation method and called this method inside `FlushFreeObjects` instead.